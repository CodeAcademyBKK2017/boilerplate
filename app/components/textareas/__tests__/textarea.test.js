import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

// Note: test renderer must be required after react-native.
import Textarea from '../textarea.component';
// import {shallow} from 'enzyme';

describe('Textarea', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Textarea />
    );
    expect(tree).toBeDefined();
  });
  it('Textarea: renders correctly', () => {
    const tree = renderer.create(<Textarea />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  
});
