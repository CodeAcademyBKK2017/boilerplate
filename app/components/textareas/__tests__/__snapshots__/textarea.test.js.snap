// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea Textarea: renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "left",
      }
    }
  >
    Content
  </Text>
  <TextInput
    allowFontScaling={true}
    multiline={true}
    style={
      Object {
        "borderColor": "gray",
        "borderRadius": 5,
        "borderWidth": 1,
        "height": 150,
        "width": 300,
      }
    }
    value=""
  />
</View>
`;
